make_level.py

```diff
@@ -0,0 +1,117 @@
+import sys
+import os
+
+cwd = os.getcwd()
+
+def write_init(skill_name, skill_path, level_name):
+    # If skills/skill/__init__.py doesn't exist, create a basic version (no levels)
+    if not os.path.exists(os.path.join(skill_path,"__init__.py")):
+        with open(os.path.join(skill_path,"__init__.py"), 'w+') as fp:
+            level_setup = "\n".join([
+                "import pkg_resources\n",
+                "from gitgud.skills.util import BasicLevel",
+                "from gitgud.skills.util import Skill\n"
+                "skill = Skill(",
+                "    '{}',".format(skill_name),
+                "    [",
+                "    ]",
+                ")\n",
+                ])
+            fp.write(level_setup)
+    
+        
+        # Read file
+        with open(os.path.join(os.path.join("gitgud","skills"),"__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        # Add import statement
+        replace = "from gitgud.skills.{0} import skill as {0}_skill\n\nfrom gitgud.skills.u".format(skill_name)
+        filedata = filedata.replace("\nfrom gitgud.skills.u", replace)
+        
+        # Add to input array of Skill
+        replace = ",\n    {}_skill\n]".format(skill_name)
+        filedata = filedata.replace("\n]", replace)
+
+        # Write to file
+        filepath = os.path.join(os.path.join("gitgud","skills"),"__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Modified: Added skill to {}".format(filepath))
+        
+        write_init(skill_name, skill_path, level_name)
+        return
+    else:
+        # Populate file with BasicLevel
+        with open(os.path.join(skill_path,"__init__.py"), 'r') as fp:
+            filedata = fp.read()
+
+        replace = ",\n        BasicLevel('{0}', pkg_resources.resource_filename(__name__, '_{0}/'))\n    ]".format(level_name)
+        filedata = filedata.replace("\n    ]", replace)
+        filedata = filedata.replace("[,","[")
+        
+        filepath = os.path.join(skill_path,"__init__.py")
+        with open(filepath, 'w') as fp:
+            fp.write(filedata)
+            print("Modified: Added level to {}".format(filepath))
+    return
+
+def create_file(level_path,filename):
+    with open(os.path.join(level_path,filename),'a+'):
+        pass
+    print("Created: {}".format(filename))
+
+def main():
+    # Obtain input arguments
+    skill_name = sys.argv[1]
+    level_name = sys.argv[2]
```

Comment:
Argument handling. try/except with error message telling correct usage

Reply:
I think I did it? New to python so I've never used this. Let me know.